chrome/service-worker.js

```
@@ -0,0 +1,128 @@
+
+  if (message.action === 'addToBanList') {
+    const { channelName } = message;
+    
+    getBanListFromStorage().then(banList => {
+      if (!banList.includes(channelName)) {
+        banList.push(channelName);
+        saveBanListToStorage(banList).then(() => {
+          // Notify all content scripts to refresh
+          chrome.tabs.query({}, (tabs) => {
+            tabs.forEach(tab => {
+              if (tab.url && tab.url.includes('youtube.com')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)